cliquet/storage/postgresql/schema.sql

```diff
@@ -26,13 +30,53 @@ CREATE TABLE IF NOT EXISTS records (
 
     PRIMARY KEY (id, parent_id, collection_id)
 );
+--
+-- CREATE INDEX IF NOT EXISTS will be available in PostgreSQL 9.5
+-- http://www.postgresql.org/docs/9.5/static/sql-createindex.html
+DO $$
+BEGIN
 
-DROP INDEX IF EXISTS idx_records_parent_id_collection_id_last_modified;
-CREATE UNIQUE INDEX idx_records_parent_id_collection_id_last_modified
+  IF NOT EXISTS (
+    SELECT 1 FROM pg_indexes
+       WHERE indexname = 'idx_records_parent_id_collection_id_last_modified'
+       AND tablename = 'records'
+  ) THEN
+
+  CREATE UNIQUE INDEX idx_records_parent_id_collection_id_last_modified
     ON records(parent_id, collection_id, last_modified DESC);
```

Comment:
>  log will be more verbose

Yes, that what I thought in my first comment :) 

This works for me ! I just wanted to challenge your choice :) 

Thanks for the details !